src/rdkafka.c

```diff
@@ -3226,10 +3275,15 @@ rd_kafka_resp_err_t rd_kafka_consumer_close(rd_kafka_t *rk) {
                         rd_kafka_op_res_t res;
```

Comment:
seems a bit messy to be working with `rkq` in the `rebalance_rkq` case. logic is fine, just probably not as clear as it could be.